tlslite/ocsp.py

```diff
@@ -0,0 +1,114 @@
+from .utils.asn1parser import ASN1Parser
+
+
+class OCSPRespStatus:
+    """ OCSP response status codes (RFC 2560) """
+    successful = 0
+    malformedRequest = 1
+    internalError = 2
+    tryLater = 3    # 4 is not used to match RFC2560 specification
+    sigRequired = 5
+    unauthorized = 6
+
+
+class CertStatus:
+    """ Certificate status in an OCSP response """
+    good, revoked, unknown = range(3)
+
+
+class SingleResponse(object):
+    """ This class represents SingleResponse ASN1 type (defined in RFC2560) """
+    def __init__(self, hashAlgorithm=None, issuerNameHash=None,
+                issuerKeyHash=None, serialNumber=None, status=None,
+                thisUpdate=None, nextUpdate=None, parser=None):
+        if parser is not None:
```

Comment:
why both `parser` as an argument to init and a `parse()` method?
Using just `parse()` would be more consistent with the rest of library.